Gems/Atom/Feature/Common/Code/Source/Debug/RayTracingDebugFeatureProcessor.h

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <Atom/Feature/Debug/RayTracingDebugFeatureProcessorInterface.h>
+#include <Atom/RPI.Public/Shader/ShaderResourceGroup.h>
+#include <Debug/RayTracingDebugSettings.h>
+
+namespace AZ::Render
+{
+    class RayTracingDebugFeatureProcessor : public RayTracingDebugFeatureProcessorInterface
```

Comment:
Quick comment about the class and the methods below.

Reply:
Added comments for class and private method (not for overwritten methods)